### DISCUSSION TITLE
Pimcore Opening Assets : “Warning: getimagesize(): corrupt JPEG data: 108 extraneous bytes before marker”

### DISCUSSION THREAD

**Discussion by manojthotla25** [pimcore/pimcore#11175] [General]
Hi Team, 

We have an issue with opening some assets in pimcore.

It is throwing the error:

“Warning: getimagesize(): corrupt JPEG data: 108 extraneous bytes before marker”

After some research in Google, we have figured out that we need to apply some specific flag in php.ini file:

but it was not resolved the issue.

Can you please provide the solution on this??]
<img width="1195" alt="Screenshot 2022-01-13 at 12 06 41 PM" src="https://user-images.githubusercontent.com/93549976/149278402-b52e5bd7-bae8-4e6c-b35d-8327310fe90a.png">

**Comment by vandanasingal** [comment]
@dvesh3 @weisswurstkanone Can you please help with this as we have a release pending due to this issue?

**Reply by fashxp** [reply]
if you need anything urgent from us, please also consider SLAs, see https://pimcore.com/en/services

**Comment by dvesh3** [comment]
Hi @manojthotla25, could you please share complete stack trace from the logs? thanks!

**Reply by manojthotla25** [reply]
Hi @dvesh3, The issue seems to be solved. Thank you for your Response!
